### PR TITLE
Enable/Disable new env config and deploy changes

### PR DESCRIPTION
**Pull request by yujunglo**
@sbaogang 
1. Disable All Envs
screen shot 2016-05-10 at 2 38 21 pm
screen shot 2016-05-10 at 2 38 44 pm

2. Message
screen shot 2016-05-10 at 2 39 01 pm

3. Disable/Enable button for each env stage
screen shot 2016-05-10 at 2 40 24 pm

### CODE REVIEW

File: deploy-board/deploy_board/templates/environs/envs_landing.html [hunk 1]
```diff
@@ -9,6 +9,116 @@
...
+     aria-labelledby="disableconfirmModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <form id="disableConfigFormId" class="form-horizontal" role="form" method="post" action="/envs/disable/">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+                <h4 class="modal-title">Disable Confirm</h4>
+            </div>
+            <div class="modal-body">
+                <p>Are you sure to limit new configuration and deploy changes for every environments?</p>
```
Comment (sbaogang) [inline deploy-board/deploy_board/templates/environs/envs_landing.html]: Are you sure to disable all new deploys and configuration changes for all environments?

File: deploy-board/deploy_board/templates/configs/env_config.html [hunk 2]
```diff
@@ -113,8 +115,75 @@ <h4 class="panel-title pull-left">Configuration</h4>
...
+     aria-labelledby="disableconfirmModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <form id="disableConfigFormId" class="form-horizontal" role="form" method="post" action="/env/{{ env.envName }}/{{ env.stageName }}/disable/">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+                <h4 class="modal-title">Disable Confirm</h4>
+            </div>
+            <div class="modal-body">
+                <p>Are you sure to limit new configuration and deploy changes for {{ env.envName }} ({{ env.stageName }})?</p>
```
Comment (sbaogang) [inline deploy-board/deploy_board/templates/configs/env_config.html]: Are you sure to disable new deploys and configuration changes for...

**Comment by sbaogang** [comment]
A minor comments on wording, otherwise looks good to me, thanks! r+

**Comment by lilida** [comment]
There will be several merge conflicts in the two pull requests. I will wait for your merge and rebase again. We need to do an integration test after both changes are in

**Pull request merged**